packages/result-app/package.json

```diff
@@ -10,23 +14,21 @@
         "@types/node": "^12.0.0",
         "@types/react": "^17.0.0",
         "@types/react-dom": "^17.0.0",
```

Comment:
version 17.0.9 was released 3 months ago.  When react was updated to 17.0.2 6 months ago, these also were updated to 17.0.2/3

Reply:
I don't see what you're saying